compiler/rustc_smir/src/rustc_smir/mod.rs

```diff
@@ -491,6 +491,10 @@ impl<'tcx> Stable<'tcx> for mir::coverage::CoverageKind {
                     rhs: opaque(rhs),
                 }
             }
+            CoverageKind::Branch { true_op, false_op } => stable_mir::mir::CoverageKind::Branch {
+                true_op: opaque(true_op),
+                false_op: opaque(false_op),
+            },
```

Comment:
Not directly related to this PR, but I was a little concerned to see that SMIR has copies of coverage-related data structures.

[I've opened a Zulip thread in #project-stable-mir](https://rust-lang.zulipchat.com/#narrow/stream/320896-project-stable-mir/topic/SMIR.20and.20StatementKind.3A.3ACoverage) to hopefully get some clarity on how SMIR and coverage should interact.